misc/conda/make_conda_env.sh

```diff
@@ -251,9 +219,16 @@ if [ "$(which dpkg 2>/dev/null)" ]; then
     fi
 fi
 
+echo
 echo "Activate the conda environment with the command"
 echo "  conda activate $ENVNM"
 echo "The environment can be deactivated with the command"
 echo "  conda deactivate"
+echo
+echo "Jax installed without GPU support. To avoid warning messages,"
```

Comment:
This means if GPU support is required then you should not use this script?

Reply:
No, it means that manual action is required *after* use of the script. I had assumed that this was implied by the following sentence "To include GPU support ...", but perhaps it's not as clear as I had thought?

Reply:
It was not clear to me. Let's make it more clear.

Also in the `.rst` file we could make it something like `To include GPU support, follow the jax installation instructions after running the installation script.`

Reply:
Done.